packages/types/src/committee.ts

```diff
@@ -2,7 +2,7 @@ import { z } from "zod"
 
 export const CommitteeSchema = z.object({
   id: z.string().ulid(),
-  createdAt: z.date(),
+  createdAt: z.coerce.date(),
```

Comment:
Why is this needed? we use superjson so dates shouldn't have to be coerced into string anywhere

Reply:
Did not mean for the latest commit to be reviewed, was mostly to show code to Anhkha (: so this was from testing earlier

Reply:
but yeah my bad, sorry

Reply:
ah no problem